Makefile

```diff
@@ -47,6 +47,7 @@ REQ_FILES = \
 	requirements/edx/paver \
 	requirements/edx-sandbox/shared \
 	requirements/edx-sandbox/base \
+	requirements/edx-sandbox/pre \
```

Comment:
Should't `/pre` be listed before `/shared` and `/base`?

Reply:
@pomegranited, the order doesn't matter here as the requirements are not being installed in this target. But I will change it to make it more obvious.

Reply:
Ah gotcha.. thanks though!